src/pages/user/authority/AuthorityOrcidModal.tsx

```diff
@@ -20,6 +21,10 @@ const StyledFooter = styled.div`
 const AuthorityOrcidModal: FC = () => {
   const { t } = useTranslation('common');
   const user = useSelector((store: RootStore) => store.user);
+  const { location } = useHistory();
+
+  const noOrcid = user.authority?.orcids.length === 0;
+  const noFeide = user.authority?.feideids.length === 0;
 
   const [doNotShowAuthorityModalAgain, setDoNotShowAuthorityModalAgain] = useState(false);
   const [doNotShowOrcidModalAgain, setDoNotShowOrcidModalAgain] = useState(false);
```

Comment:
siden du sa at denne komponenten hadde my stuff: trengs disse to? kan man ikke bare bruke variablene fra LS under i stedet?

Reply:
nvm, ser lenger ned at disse er current state, mens data i LS er for senere